openeogeotrellis/job_options.py

```diff
@@ -48,15 +48,17 @@ class JobOptions:
         default=get_backend_config().default_executor_memoryOverhead,
         metadata={
             "name": "executor-memoryOverhead",
-            "description": "Memory allocated to the workers in addition to the JVM, for example, to run UDFs. "
-            "The total available memory of an executor is equal to executor-memory + executor-memoryOverhead [+ python-memory].",
+            "description": "Memory allocated to the workers in addition to the JVM. "
+            "The total available memory of an executor is equal to executor-memory + executor-memoryOverhead + python-memory.",
         })
     python_memory: str = field(
         default=get_backend_config().default_python_memory,
         metadata={
             "description": "Setting to specifically limit the memory used by python on a worker. "
+            "Leaving this setting empty will enforce the default value of the backend. None will be zero bytes."
```

Comment:
these lines are appended as is I think, so you have to put spaces at end (unless last sentence):
```suggestion
            "Leaving this setting empty will enforce the default value of the backend. None will be zero bytes. "
```